src/pybind/mgr/zabbix/module.py

```diff
@@ -186,40 +186,42 @@ def get_data(self):
         data['num_mon'] = len(mon_status['monmap']['mons'])
 
         df = self.get('df')
-        data['num_pools'] = len(df['pools'])
-        data['total_used_bytes'] = df['stats']['total_used_bytes']
-        data['total_bytes'] = df['stats']['total_bytes']
-        data['total_avail_bytes'] = df['stats']['total_avail_bytes']
+        data['num_pools'] = len(df.get('pools'))
```

Comment:
this change would mean that we would fail silently, if these elements vanish again. 

I'm :+1: for keeping the existing behaviour that raises exceptions, if any keys get changed again.

Reply:
I'm ok with exceptions too - just as I'm not really aware we're tests are being runned (as they're not failing during PR check with Jenkins), decided, that it might be useful to ignore them. Will return previous behavior then